packages/core/src/internal/indicator-container/indicator-container.scss

```
@@ -82,7 +83,8 @@
     > .ods-checkbox:checked,
     > .ods-checkbox:indeterminate,
     > .ods-radio:checked {
-      &:not(:disabled):not(.-invalid) + .ods-input-indicator {
+      &:not(.-touched:invalid:disabled) + .ods-input-indicator,
+      &:not(.-invalid:disabled) + .ods-input-indicator {
```

Minor, however to follow other changed selectors:

```suggestion
      &:not(.-invalid:disabled) + .ods-input-indicator,
      &:not(.-touched:invalid:disabled) + .ods-input-indicator {
```